cljfmt/src/cljfmt/main.clj

```diff
@@ -167,7 +167,7 @@
   (or (find-file-in-dir dir ".cljfmt.edn")
       (find-file-in-dir dir ".cljfmt.clj")))
 
-(defn- load-configuration []
+(defn load-configuration []
```

Comment:
Since this is a public function, let's give it a docstring. Something like:

```clojure
(defn load-configuration []
  "Load a `.cljfmt.edn` or `.cljfmt.clj` configuration file from the current
  directory or from the first parent directory to contain one."
  ...)
```